### ISSUE TITLE
Error modifying asset properties

### ISSUE THREAD

**Issue by tnulc** [Bynder/bynder-python-sdk#10]
Calling
```python
asset_bank_client.set_media_properties('715AB17A-AE69-4B40-AFA7C1295A874B89', {'metaproperty.1389E77E-F6A1-4698-95DC07D30C6E1683': '8E92E068-E335-425C-805D16E8D04026AA'})
```

Results in:

```
"/Users/christopherlunt/.pyenv/versions/imageaudit/lib/python3.7/site-packages/bynder_sdk/client/asset_bank_client.py", line 53, in set_media_properties
    payload=query or {}
  File "/Users/christopherlunt/.pyenv/versions/imageaudit/lib/python3.7/site-packages/bynder_sdk/util.py", line 27, in post
    return self.wrapped_request(super().post, url, *args, **kwargs)
  File "/Users/christopherlunt/.pyenv/versions/imageaudit/lib/python3.7/site-packages/bynder_sdk/util.py", line 16, in wrapped_request
    response = func(endpoint, *args, **kwargs)
  File "/Users/christopherlunt/.pyenv/versions/imageaudit/lib/python3.7/site-packages/requests/sessions.py", line 581, in post
    return self.request('POST', url, data=data, json=json, **kwargs)
TypeError: request() got an unexpected keyword argument 'payload'
```

I can't seem to modify any properties, even calling with an empty query causes an error.

**Comment by danielsequeira** [comment]
Solved with https://github.com/Bynder/bynder-python-sdk/pull/11

**Comment by WouterToering** [comment]
Thanks for reporting @kirisu, I've just released 1.1.1 with a fix. Could you please try again and let us know if everything is working as expected now?

(PR got autoclosed by the merged commit)

**Comment by tnulc** [comment]
Hi @WouterToering, can confirm this now works 👍. Thanks for the quick response and fix.
